docs/index.html

```diff
@@ -10,6 +10,19 @@
       <script src="https://unpkg.com/vue-async-computed@3.8.1"></script>
       <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.0.0/highlight.min.js"></script>
       <script src="https://cdnjs.cloudflare.com/ajax/libs/axios/0.18.0/axios.min.js"></script>
+      <script>
+        function hideHeaderIfSearching() {
+          var x = document.getElementById("search");
+
+          var myDiv = document.getElementById("about");
+          if (x.value != "")
+          {
+              myDiv.style.display = "none";
+          } else {
+              myDiv.style.display = "block";
+          }
```

Comment:
Given that we're using vue.js on the page. I think it would be better to try to manage all the state in vue. You could add the following to the `asyncComputed` section:

```javascript
asyncComputed: {
  async isSearching() {
    // returns true if this.searchCondition contains text, false otherwise.
    return !this.searchCondition.trim()
  },
  // additional async functions ..
}
```

And then update the `<div v-html="aboutHtml"></div>` below as follows:

```diff
-<div v-html="aboutHtml" id="about"></div>
+<div v-if="isSearching" v-html="aboutHtml"></div>
```

We also wouldn't need the `onKeyup="hideHeaderIfSearching()"` when making the change.